test/testme_logger_test.cpp

```diff
@@ -0,0 +1,70 @@
+#include <format>
+#include <gtest/gtest.h>
+#include <string>
+#include <testme_logger/logger.hpp>
+
+TEST(LOGGER_SERVICE_TEST, TestInfoMessageStringIsCorrectCout) {
```

Comment:
Need more unit tests! Need to check the other labels as well!!! [WARNING] [ERROR] etc etc

Also need unit test for checking log file content.